mkdocs.yml

```diff
@@ -60,8 +61,11 @@ edit_uri: edit/main/docs/
 
 markdown_extensions:
   - admonition
+  - abbr
   - def_list
   - footnotes
+  - attr_list
+  - pymdownx.extra
```

Comment:
I couldn't find where these extensions are being used. Can you show me? :)

Reply:
Yes, I was doubting to add a comment, but we've not done that before.

It's used for the tabbed content and the html in markdown. To be honest these extensions are a bit weird to me.